src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import CheckIcon from '@material-ui/icons/Check';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+type sublabel = {
+  label: string;
+  amount: number;
+};
+
+interface HomeMenuItemProps {
+  label: string;
+  amount?: number;
```

Comment:
Maybe `amount` isn't the best variable name here / for sublabel (or maybe the component itself is named weird? I think it's odd that a component named `HomeMenuItem` takes in an `amount` prop to begin with. I can't think of a better off the top of my head but I will keep thinking. If you can't think of a better name either we could also just put this on the backburner as well (or just deal with the weird naming)

Reply:
also agree that `amount` doesn't seem to fit, but not sure what else would be better. thoughts on `alerts` or `itemAlerts`? also not in love with those ideas either though

Reply:
I can make a note of it and keep thinking, but this shouldn't block. the PR from merging. I approved, thank you :)